src/components/network/network-table.js

```diff
@@ -94,6 +112,37 @@ const NetworkTable = (props) => {
         [lineEdit, tabIndex]
     );
 
+    useEffect(() => {
+        fetchConfigParameter(
+            COLUMNS_PARAMETER_PREFIX_IN_DATABASE + TABLES_NAMES[tabIndex]
+        ).then((val) => {
+            console.debug('received UI parameter : ', val);
+            if (!val) {
+                setSelectedColumnsNames(TABLES_COLUMNS_NAMES[tabIndex]);
+            } else {
+                setSelectedColumnsNames(new Set(JSON.parse(val.value)));
+            }
+        });
+    }, [tabIndex]);
+
+    useEffect(() => {
+        if (selectedColumnsNames && notifiedDisplayedColumns) {
+            let notifiedTableName = notifiedDisplayedColumns.name.slice(
+                COLUMNS_PARAMETER_PREFIX_IN_DATABASE.length
+            );
+            if (
+                TABLES_NAMES.indexOf(notifiedTableName) === tabIndex &&
+                JSON.stringify([...selectedColumnsNames]) !==
+                    notifiedDisplayedColumns.value
+            ) {
+                setSelectedColumnsNames(
+                    new Set(JSON.parse(notifiedDisplayedColumns.value))
+                );
+            }
+            dispatch(selectDisplayedColumns(null));
```

Comment:
it's weird to reset the redux state to null here. If redux states becomes the source of truth instead of the local copied state, this dispatch must be removed

Reply:
It is a hybrid (redux/local) solution to have a local state.
I will try to implement a solution full redux.